salt/_modules/metalk8s_solutions.py

```diff
@@ -1,186 +1,203 @@
-'''
-Various utilities to manage Solutions.
-'''
-import json
+"""Utility methods for Solutions management.
+
+This module contains minion-local operations, see `metalk8s_solutions_k8s.py`
+for the K8s operations in the virtual `metalk8s_solutions` module.
+"""
+import collections
+import errno
 import logging
-import yaml
 
 from salt.exceptions import CommandExecutionError
-
-HAS_LIBS = True
-SOLUTIONS_CONFIG_MAP = 'metalk8s-solutions'
-SOLUTIONS_CONFIG_MAP_NAMESPACE = 'metalk8s-solutions'
-SOLUTIONS_CONFIG_FILE = '/etc/metalk8s/solutions.yaml'
-try:
-    import kubernetes.client
-    from kubernetes.client.rest import ApiException
-    from urllib3.exceptions import HTTPError
-except ImportError:
-    HAS_LIBS = False
+import yaml
```

Comment:
Wut?! As your linter (`pylint` I guess) says, `yaml` is third party, not standard lib. So I put them together, and since `Y > S`, I put it after `salt`. Now, maybe you have a `pylint` plugin that wants `salt` imports after those, but I'm quite unsure as to why that would be a good idea.

Reply:
Oh yes indeed I use saltpylint. 
https://docs.saltstack.com/en/latest/topics/development/conventions/style.html#salt-coding-style

So yes it's 
```
# import builtins

# import thirdparty

# import salt_stuff
```